src/garage/torch/policies/base.py

```diff
@@ -35,3 +35,8 @@ def observation_space(self):
     def action_space(self):
         """Policy action space."""
         return self._env_spec.action_space
+
+    @property
+    def vectorized(self):
```

Comment:
I think we're just going to delete the method, and assume that all policies are vectorized (i.e. that all policies implement `get_actions`). You can make this change in a later PR though.

Reply:
I tried to but I checked that `BatchPolopt` use the `vectorized` property to decide `sampler_cls`
[(code)](https://github.com/rlworkgroup/garage/blob/7bb784efdefc7fd8f160718c1d989144ba91e07f/src/garage/np/algos/batch_polopt.py#L40)
Do I need to change the `BatchPolopt`?

Reply:
You don't need to change `BatchPolopt` in this PR, but it would be good for this function to always return True, and have a default implementation of `get_actions` in terms of `get_action`.

Once every policy in the code base is always vectorized, then we'll modify `BatchPolopt`.

Reply:
So is the right thing to do here to return `True`?

Reply:
Yeah, this method should return `True`, and `get_actions` should be implemented in terms of `get_action`.